pandas/tests/extension/test_categorical.py

```diff
@@ -192,6 +192,15 @@ def test_searchsorted(self, data_for_sorting):
         if not data_for_sorting.ordered:
             raise pytest.skip(reason="searchsorted requires ordered data.")
 
+    @pytest.mark.parametrize(
```

Comment:
why is this not in the base tests? are you testing ordered and not-ordered Categoricals?

Reply:
Due to #30227, the empty categorical array doesn't raise `ValueError` but returns `nan`. So `max` and `min` are excluded from this test.